Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -405,6 +409,10 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         Animate {
             let leadingContainerSubviews = self.composerView.leadingContainer.subviews
             for subview in leadingContainerSubviews where subview !== self.composerView.shrinkInputButton {
+                let channelConfig = self.channelController.channel?.config
+                if subview is AttachmentButton, channelConfig?.uploadsEnabled == false {
+                    continue
+                }
```

Comment:
Comparing to `AttachmentButton` will fail for custom UIButton subclasses. We should probably check if it's the actual button instance for the attachment button.

Reply:
Yes you are totally right 🤦‍♂️